src/compiler/sys/in-memory-fs.ts

```diff
@@ -719,13 +1072,26 @@ export const createInMemoryFs = (sys: d.CompilerSystem) => {
     return item;
   };
 
-  const clearCache = () => items.clear();
-
-  const keys = () => Array.from(items.keys()).sort();
```

Comment:
I could find no references to these functions in the codebase, so I think they should be safe to delete